src/state/state_writer.rs

```diff
@@ -298,8 +298,8 @@ mod tests {
 
     #[test]
     fn test_multi_thread() {
-        let outer_loop_iterations = 1000;
-        let inner_loop_iteration = 2000;
+        let outer_loop_iterations = 500;
```

Comment:
Maybe, it would be better to add a comment on this line for the future.

Reply:
Done.